src/App.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+import accents from 'remove-accents';
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '@fortawesome/fontawesome-free/css/all.min.css';
+
+import './App.scss';
+
+import Search from './components/Search/Search';
+import Word from './components/Word/Word';
+import GithubButtons from './components/GithubButtons/GithubButtons';
+import Header from './components/Header/Header';
+
+import words from './data.json';
+
+const App: React.FC = function App() {
+  const [filteredWords, setFilteredWords] = useState<Word[]>(words);
+  const [currentWord, setCurrentWord] = useState<string>('');
```

Comment:
For this case (basic types) it's preferable rely on type inference.

Reply:
good catch! on it 🏃🏽‍♂️